slack_sdk/models/blocks/block_elements.py

```diff
@@ -1081,19 +1369,28 @@ def __init__(
         self,
         *,
         action_id: Optional[str] = None,
-        placeholder: Optional[Union[str, dict, TextObject]] = None,
```

Comment:
Radio buttons block element does not accept placeholder. This has been wrong for a long time.